code/modules/reagents/reagents/food.dm

```diff
@@ -102,35 +102,35 @@
 
 /datum/reagent/consumable/protein/mint
 	name = "Mint Protein Mix"
-	taste_description = "minty protein"
+	//taste_description = "minty protein"
 
 /datum/reagent/consumable/protein/grape
 	name = "Grape Protein Mix"
```

Comment:
Commented out code

Reply:
ill remove it once i fix flavor still workin